helpers/apiCodes.js

```diff
@@ -2,6 +2,8 @@
 
 module.exports = {
 	SUCCESS: 200,
-	EMPTY_RESOURCES_SUCCESS: 204,
-	ERROR_DEFAULT: 500
+	EMPTY_RESOURCES_SUCCESS: 200,
+	ERROR_DEFAULT: 500,
+	QUERY_MALFORMED: 400,
```

Comment:
I would call this variables as standard as possible:

200 OK
400 Bad request / Client error
429 Too Many Requests 
500 Internal Server Error

Reference from IANA http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml

Reply:
Will rename it, thanks